src/Infrastructure/LeanCode.ForceUpdate/LeanCode.ForceUpdate.Services/VersionHandler.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Http;
+
+namespace LeanCode.ForceUpdate.Services;
+
+public class VersionHandler
+{
+    public virtual VersionSupport CheckVersion(
```

Comment:
Actually, I can probably think of some cases where overrides of this method would want to perform some async call (e.g. we want to force update for specific tenants only and we store that information in the database). Can we make it `virtual Task<VersionSupport> CheckVersionAsync` instead?

(Aaand that makes my previous suggestion about refactoring `Task.FromResult` no longer valid, sorry :sweat_smile:)